changelog.md

```diff
@@ -1,5 +1,28 @@
 
 
+## 2.7.0 / 2020-04-24
+
+### New Features
+
+- Support Português language
+
+### Improvement
+
+- Better Tracim reactivity by improving SQL requests (#697)
+
+### Fixed Issues
+
+- General UX: #1409, #1444, #1515, #1855, #2406, #2444, #2454, #2537, #2595, #2661, #2673, #2679, #2681, #2693, #2701, #2764, #2791
+- Frontend: #1613, #2569
+- Backend: #941, #2631, #2737
+- Documentation: #2776
+- Docker: #2753
+
+### Other Changes
+
+- The configuration file (development.ini.sample) now contains the default values used by Tracim and explanations on the settings (#2204). We advise you to be careful on existing installations when merging your configuration file with the new one. 
+
```

Comment:
@PhilippeAccorsi You should also notify breaking change about database compatibility. We do not support anymore old mysq/mariadb version.

Reply:
In backend Readme:
"If you want to use PostgreSQL(9.3+), MySQL(8.0.1+) or MariaDB(10.3+) database engine instead of the default one (SQLite bundle with python)."

Reply:
Right, I need to add breakingchnage information.

Reply:
Done